deploy/dev/config.sample.yaml

```diff
@@ -8,6 +8,7 @@ entra:
   tenant_id: <CHANGE_ME>
```

Comment:
could you update the comment above this line with the additional permissions needed for the invite to work